src/main/java/com/google/graphgeckos/dashboard/storage/ParsedBuildbotData.java

```diff
@@ -59,16 +59,6 @@ boolean validUpdateData() {
     return (commitHash != null && builderName != null && logs != null && status != null);
   }
 
-  /**
-   * Converts this instance to an instance of
-   * {@link com.gogle.graphgeckos.dashboard.storage.Builder Builder}.
-   *
-   * @return a new Builder instance, with the fields copied from the calling instance.
-   */
-  Builder toBuilder() {
-    return new Builder(builderName, logs, status);
-  }
-
   /**
    * Getter for the commitHash.
```

Comment:
I would suggest changing the comment to ```Returns commitHash``` and remove the ```Getter for ...``` part.

Reply:
This isn't in the scope of this PR. I will update all the POJOs documentation to be in-line with the newer ones, once the storage component is up and running